<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Harshita4452.local" name="com.harshita.selenium.testcases.HomePageTest" tests="3" failures="1" timestamp="2021-10-13T08:58:50 IST" time="92.960" errors="0">
  <testcase name="TC_02" time="50.689" classname="com.harshita.selenium.testcases.HomePageTest"/>
  <system-out/>
  <testcase name="TC_01" time="16.396" classname="com.harshita.selenium.testcases.HomePageTest"/>
  <system-out/>
  <testcase name="TC_01" time="25.875" classname="com.harshita.selenium.testcases.HomePageTest">
    <failure type="java.lang.AssertionError" message="expected [Email should be present in the account] but found [There is no email in the account]">
      <![CDATA[java.lang.AssertionError: expected [Email should be present in the account] but found [There is no email in the account]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.harshita.selenium.testcases.HomePageTest.TC_01(HomePageTest.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:45)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:73)
at org.testng.internal.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- TC_01 -->
  <system-out/>
</testsuite> <!-- com.harshita.selenium.testcases.HomePageTest -->
